androidClient/app/src/main/java/com/csatimes/dojma/activities/AboutUsActivity.java

```diff
@@ -29,20 +32,44 @@
 
     public static String ABOUT_US_FACEBOOK_URL = "https://www.facebook.com/MACBITSGoa";
     public static String ABOUT_US_FACEBOOK_PAGE_ID = "MACBITSGoa";
+    public static String ABOUT_US_GITHUB_URL = "https://github.com/MobileApplicationsClub";
+    public static String ABOUT_US_LINKEDIN_URL = "https://www.linkedin.com/mwlite/company/13598216";
+    public static String ABOUT_US_WEBSITE_URL = "https://macbitsgoa.com";
+    public static String ABOUT_US_GOOGLEPLAY_URL = "https://play.google.com/store/search?q=Mobile%20App%20Club%20-%20BITS%20Goa&c=apps&hl=en";
+    private Context context = AboutUsActivity.this;
+    private Intent copy_intent = new Intent(context, CopyLinkBroadcastReceiver.class);
+    private PendingIntent copy_pendingIntent = PendingIntent.getBroadcast(context, 0, copy_intent, PendingIntent.FLAG_UPDATE_CURRENT);
+    private String copy_label = "Copy Link";
+
+    private int colorResource = getChromeCustomTabColorFromTheme();
+    private final CustomTabsIntent customTabsIntent = new CustomTabsIntent.Builder()
+            .setShowTitle(true)
+            .setToolbarColor(colorResource)
+            .setCloseButtonIcon(BitmapFactory.decodeResource(context.getResources(), R.drawable.ic_arrow_back_white_24dp))
+            .addMenuItem(copy_label, copy_pendingIntent)
+            .addDefaultShareMenuItem()
+            .enableUrlBarHiding()
+            .build();
+
 
     @Override
     protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_about_us);
 
-        Toolbar toolbar = (Toolbar) findViewById(R.id.about_us_toolbar);
-        ImageButton facebookImgBtn = (ImageButton) findViewById(R.id.content_about_us_fb_imgbtn);
-        ImageButton googlePlayImgBtn = (ImageButton) findViewById(R.id.content_about_us_google_play_imgbtn);
-        androidx.recyclerview.widget.RecyclerView contributorsRecyclerView = (RecyclerView) findViewById(R.id.content_about_us_rv);
+        Toolbar toolbar = findViewById(R.id.about_us_toolbar);
```

Comment:
keep all links in a single rv, each item will have icon url, title, and link to open on click. All of these from firebase